app/experimenter/static/js/designForm.js

```diff
@@ -107,13 +108,29 @@ export default class DesignForm extends React.Component {
     const res = await this.makeFetchCall("PUT", JSON.stringify(object));
 
     if (res.status == "200") {
-      location.replace(`/experiments/${this.props.slug}/`);
+      if (this.state.continue){
+        location.replace(`/experiments/${this.props.slug}/edit-objectives/`);
+      }
+      else{
+        location.replace(`/experiments/${this.props.slug}/`);
+      }
+      
     }
 
```

Comment:
Just thinking.... would it better for me to reset continue back to false here?

Reply:
Well, try to always remember my personal golden rule of react: if you can avoid putting something in state, don't put it in state.  When the user clicks either `Save` or `Save and Continue` and from that point on it's very unidirectional, we call a method, make a fetch, and if it's valid then we redirect.  So do we need to put it in state at all?  Could we just have two separate methods for submitting?